test/functional/test_framework/wallet_util.py

```diff
@@ -88,6 +88,11 @@ def get_multisig(node):
                     p2sh_p2wsh_script=CScript([OP_HASH160, witness_script, OP_EQUAL]).hex(),
                     p2sh_p2wsh_addr=script_to_p2sh_p2wsh(script_code))
 
+def labels_value(name="", purpose="receive"):
```

Comment:
Question: Would it make sense for the `labels_value` method to indicate that it is only for the single member `labels` case (since `getaddressinfo` may support multiple labels in the future)?

Reply:
Thanks @jachiang. I thought I'd update it if/when adding support for multiple labels, though happy to take a good suggestion now for a better name or docstring.

Reply:
You're right, a docstring or inline comment would have probably sufficient for sb who is just looking at the functional test, and not the rpc code (where labels vector is actually documented). Happy to wait and see what other reviewers think :)

Reply:
The rpc code doc seems good, but explaining it in the help might be a good idea. Thanks, might update that. Thanks for the review!